desk/app/chat.hoon

```diff
@@ -1078,27 +1079,40 @@
       %+  ~(put by reply-index)  u.parent-time
       (put:on-v-replies:d old-replies time `(convert-quip time writ))
     %+  gas:on-v-posts:d  *v-posts:d
-    %+  murn  writs
-    |=  [=time =writ:t]
-    ^-  (unit [id-post:d (unit v-post:d)])
-    ?^  replying.writ  ~
+    =|  posts=(list [id-post:d (unit v-post:d)])
+    =<  +
+    %+  roll  writs
+    |=  [[=time =writ:t] count=@ud =_posts]
```

Comment:
Shouldn't define `posts` outside of the scope where it's being used. (It's not the same posts as the one you use below, anyway.) Confusing.

Consider the following, noting that the faces used in the sample type remain accessible after the accumulation product comes out of `+roll`.

```suggestion
    =<  posts
    %+  roll  writs
    |=  $:  [=time =writ:t]
            [count=@ud posts=(list [id-post:d (unit v-post:d)])]
        ==
```

Same thing in diary and heap.